app/uk/gov/hmrc/helptosave/connectors/HelpToSaveProxyConnector.scala

```diff
@@ -142,6 +144,10 @@ class HelpToSaveProxyConnectorImpl @Inject() (http:              HttpClient,
             val _ = timerContext.stop()
             response.status match {
               case Status.OK ⇒
+
+                val path = s"/help-to-save/$nino/account?nino=$nino&systemId=$systemId&correlationId=$correlationId"
```

Comment:
note that this is the url of the incoming request but not the NSI get account url

Reply:
it'll be more futureproof to pass the path in and get it from the `request` in the controller - you can find the full URI including the query parameters under `request.uri`

Reply:
fixed